.gitignore

```diff
@@ -1,8 +1,10 @@
 **/target
 **/build
 **.idea
+**.vscode
 **.iml
 /node_modules
 /.pnp
 .pnp.js
 pnpm-debug.log*
+.env
```

Comment:
Husk I meant:
1. keep .env as a sample file
2. document the user copying .env to .env.local
3. Load from .env.local

This was .env is reserved in git as an example

Reply:
wrong